src/daemon/daemon.cpp

```diff
@@ -373,6 +385,12 @@ std::vector<mp::NetworkInterface> validate_extra_interfaces(const mp::LaunchRequ
 
     if (!request->network_options().empty())
     {
+        std::string full_name =
+            (request->remote_name().empty() ? "release" : request->remote_name()) + ':' + request->image();
+
+        if (no_bridging_images.find(full_name) != no_bridging_images.end())
+            throw std::runtime_error(fmt::format("--network not implemented for {}", full_name));
```

Comment:
I just realized we should not refer to a CLI specific thing like `--network` from the daemon, since it may be meaningless in other clients (e.g. future GUI or some other thing using the GRPC).

Would you mind replacing with "extra networks" or similar? Here and below (line 401). I think this is one of those cases where it makes sense to change in the same PR, for consistency, but if you prefer, we can do it in a separate PR.

Reply:
I prefer to do it here, if @Saviq is ok with that. Not the cleanest, but we don't want to delay releasing this anymore.

Reply:
Done.